L1Trigger/L1TGlobal/interface/GlobalBoard.h

```diff
@@ -164,6 +165,32 @@ namespace l1t {
     /// pointer to Tau data list
     inline const BXVector<const GlobalExtBlk*>* getCandL1External() const { return m_candL1External; }
 
+    //initializer prescale counter using a semi-random value between [1, prescale value]
+    static const std::vector<double> semirandomNumber(const edm::Event& iEvent,
```

Comment:
I wanted to stress that this is a standalone function, that might be used even outside a GlobalBoard object.
I might define it as an external function eg. `l1t::semirandomNumber(...)`, but I don't know in which file I should define this function. I've just learned that I cannot define it as `l1t::GlobalBoard::semirandomNumber(...)` because a namespace cannot have the same name of a class.
I don't have a strong opinion. What do you suggest?

Reply:
No strong opinion here, either. :)

I guess `l1t::GlobalBoard::semirandomNumber(..)` is effectively how this function would be called outside this class if using the current implementation. I don't see a good place for this elsewhere in this package, so unless L1T has a better idea, I would leave it here like this (maybe the implementation could be moved to the `.cc` file, though).